### PR TITLE
Ankit | test | Prod - Invalid date error is coming when changing the entry date on ledger for imported transaction 

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cw4245t

### CODE REVIEW

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts [hunk 1]
```diff
@@ -95,15 +97,19 @@ export class GiddhDatepickerComponent implements ControlValueAccessor, OnInit, O
         this.destroyed$.next(true);
         this.destroyed$.complete();
     }
-
+    public dateInputChange(event: Event): void {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts:106]: js doc

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts:106]: done

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts [hunk 2]
```diff
@@ -97,11 +97,17 @@ export class GiddhDatepickerComponent implements ControlValueAccessor, OnInit, O
         this.destroyed$.next(true);
         this.destroyed$.complete();
     }
+
+    /**
+     *
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts:103]: Get input value

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts [hunk 3]
```diff
@@ -96,14 +98,24 @@ export class GiddhDatepickerComponent implements ControlValueAccessor, OnInit, O
...
     }
 
+    /**
+     * 
+     * @param event 
+     */
+    public dateInputChange(event: Event): void {
+        const inputElement = event.target as HTMLInputElement;
+        const inputValue = inputElement.value;
+        this.inputChange= inputValue;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts]:
```suggestion
        this.inputChange = inputValue;
```

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts]: done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**